pkg/scheduler/util/utils.go

```diff
@@ -163,3 +166,20 @@ func ClearNominatedNodeName(cs kubernetes.Interface, pods ...*v1.Pod) utilerrors
 	}
 	return utilerrors.NewAggregate(errs)
 }
+
+// NewInformerFactory creates a SharedInformerFactory and initializes a scheduler specific
+// in-place podInformer.
+func NewInformerFactory(cs kubernetes.Interface, resyncPeriod time.Duration) informers.SharedInformerFactory {
```

Comment:
can we keep this in the scheduler pkg? we are close to having a staging repo for common utility functions, so this pkg will very likely be removed in the 1.20.

Reply:
Sure. Updated.